package.json

```diff
@@ -36,40 +33,32 @@
     "!dist-mjs/**/*.spec.mjs",
     "SECURITY.md"
   ],
-  "devDependencies": {
-    "@checkdigit/eslint-config": "^9.2.0",
-    "@checkdigit/jest-config": "^6.0.0",
-    "@checkdigit/prettier-config": "^5.3.0",
-    "@checkdigit/typescript-config": "^7.0.1",
-    "@js-temporal/polyfill": "^0.4.4",
-    "rimraf": "^5.0.5"
-  },
   "scripts": {
-    "prepublishOnly": "npm run build:dist-types && npm run build:dist-mjs",
-    "build:dist-types": "rimraf dist-types && npx builder --type=types --outDir=dist-types",
     "build:dist-mjs": "rimraf dist-mjs && npx builder --type=module --sourceMap --outDir=dist-mjs && node dist-mjs/index.mjs",
-    "lint": "eslint --max-warnings 0 --ignore-path .gitignore .",
-    "lint:fix": "eslint --ignore-path .gitignore . --fix",
-    "prettier": "prettier --ignore-path .gitignore --list-different .",
-    "prettier:fix": "prettier --ignore-path .gitignore --write .",
-    "test": "npm run ci:compile && npm run ci:test && npm run ci:lint && npm run ci:style",
+    "build:dist-types": "rimraf dist-types && npx builder --type=types --outDir=dist-types",
     "ci:compile": "tsc --noEmit",
-    "ci:test": "NODE_OPTIONS=\"--disable-warning ExperimentalWarning --experimental-vm-modules\" jest --coverage=false",
-    "ci:coverage": "NODE_OPTIONS=\"--disable-warning ExperimentalWarning --experimental-vm-modules\" jest --coverage=true",
+    "ci:coverage": "rimraf coverage && mkdir coverage && node --env-file=.env --disable-warning ExperimentalWarning --experimental-strip-types --test-timeout 600000 --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=coverage/lcov.info --test \"src/**/*.spec.ts\"",
     "ci:lint": "npm run lint",
-    "ci:style": "npm run prettier"
-  },
-  "jest": {
-    "preset": "@checkdigit/jest-config"
+    "ci:style": "npm run prettier",
+    "ci:test": "node --env-file=.env --disable-warning ExperimentalWarning --experimental-strip-types --test-timeout 600000 --test \"src/**/*.spec.ts\"",
```

Comment:
What's different about this repo that needed `--env-file=.env`?

Reply:
it works the same way as process.loadEnvFile(), but to be consistent i agree it's better to do process.loadEnvFile explicitly in test suites.